keps/sig-node/NNNN-memory-manager-hugepages-verification/README.md

```diff
@@ -0,0 +1,478 @@
+# KEP-NNNN: Memory Manager Hugepages Availability Verification
```

Comment:
the KEP process is to file a issue on github.com/kubernetes/enhancements and use the issue number as KEP number. Example: https://github.com/kubernetes/enhancements/issues/5573

Reply:
Will do - I'll file an enhancement issue and update the KEP number accordingly.